rockhound/slab2.py

```diff
@@ -0,0 +1,147 @@
+            - `kamchatka`: Kamchatka-Kuril Islands-Japan
+            - `kermadec`: Kermadec
+            - `makran`: Makran
+            - `manila_trench`: Manila Trench
+            - `muertos_trough`: Muertos Trough
+            - `new_guinea`: New Guinea
+            - `pamir`: Pamir
+            - `philippines`: Philippines
+            - `puysegur`: Puysegur
+            - `ryukyu`: Ryukyu
+            - `scotia_sea`: Scotia Sea
+            - `solomon_islands`: Solomon Islands
+            - `south_america`: South America
+            - `sulawesi`: Sulawesi
+            - `sumatra_java`: Sumatra-Java
+            - `vanuatu`: Vanuatu
+    load : bool
+        Wether to load the data into an :class:`xarray.Dataset` or just return the
+        path to the downloaded data. If False, will return a list with the paths to the
+        subduction grids, respectively.
+    kwargs
+        Keyword arguments will be forwarded to the :func:`xarray.open_dataset` function
+        that loads the grid into memory.
+
+    Returns
+    -------
+    grid : :class:`xarray.Dataset` or str
+        The loaded grid or the file path to the downloaded data.
+    """
+    if zone not in ZONES:
+        raise ValueError("Invalid slab zone: {}".format(zone))
+    fnames = [
+        REGISTRY.fetch(
+            "{}_slab2_{}.grd".format(ZONES[zone]["fname_indicator"], dataset)
+        )
+        for dataset in DATASETS
+    ]
+    if not load:
+        return fnames
+    arrays = [xr.open_dataarray(f).rename(x="longitude", y="latitude") for f in fnames]
+    for array, dataset in zip(arrays, DATASETS):
+        array.name = dataset
+        # Change long_name and add units of each array
+        array.attrs["long_name"] = DATASETS[dataset]["name"]
+        array.attrs["units"] = DATASETS[dataset]["units"]
+    # Merge arrays into a single xr.Dataset
+    grid = xr.merge(arrays)
+    # Change units of thickness, depth and depth_uncertainty to meters
+    grid["thickness"] *= 1000
+    grid["depth"] *= 1000
+    grid["depth_uncertainty"] *= 1000
+    # Change long_name and units of longitude and latitude coords
+    grid.longitude.attrs["long_name"] = "Longitude"
+    grid.longitude.attrs["units"] = "degrees"
+    grid.latitude.attrs["long_name"] = "Latitude"
+    grid.latitude.attrs["units"] = "degrees"
+    # Add attributes to the xr.Dataset
+    grid.attrs.update(
+        {
+            "title": "Slab2 model - Zone: {}".format(ZONES[zone]["name"]),
+            "zone": zone,
+            "zone_full_name": ZONES[zone]["name"],
+            "datum": "WGS84",
+            "doi": "10.5066/F7PV6JNV",
+        }
+    )
+    # Change unit of  actual_range attribute
+    grid.depth.attrs["actual_range"] = grid.depth.attrs["actual_range"] * 1000
+    grid.thickness.attrs["actual_range"] = grid.thickness.attrs["actual_range"] * 1000
+    grid.depth_uncertainty.attrs["actual_range"] = (
+        grid.depth_uncertainty.attrs["actual_range"] * 1000
```

Comment:
This should be on the same place where the other units are changed, just to improve readability.